shared/elasticsearch-document-adapter/src/cdtnDocuments.ts

```diff
@@ -1,6 +1,7 @@
 import type {
   AgreementDoc,
-  ContributionCompleteDoc,
+  ContributionGenericDoc,
+  ContributionWithCCDoc,
```

Comment:
```suggestion
  ContributionWithAgreementDoc,
```

Pour une meilleur lecture et homogénéisation des termes

Reply:
Après si on a générique, on peut utiliser le terme Personnalisé que le métier utilise pas mal -> `ContributionCustomizedDoc` ou j'aurais mis plutôt `CustomizedContributionDoc` et `GenericContributionDoc`.

Reply:
Ah oui les 2 sont bien